src/components/date-range/date-range.test.tsx

```diff
@@ -0,0 +1,1125 @@
+import React from "react";
+import { render, act, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import DateRange, { DateRangeChangeEvent } from "./date-range.component";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+import CarbonProvider from "../carbon-provider";
+
+testStyledSystemMargin((props) => (
+  <DateRange
+    value={["10/10/2016", "11/11/2016"]}
+    onChange={() => {}}
+    {...props}
+  />
+));
+
+test("should update the input values to match the 'en-GB' locale's format when initialised with valid ISO strings", () => {
+  render(
+    <DateRange
+      startLabel="start"
+      endLabel="end"
+      value={["2016-10-10", "2016-11-11"]}
+      onChange={() => {}}
+    />
+  );
+
+  expect(screen.getByRole("textbox", { name: "start" })).toHaveValue(
```

Comment:
```suggestion
  expect(screen.getByLabelText("start")).toHaveValue(
```

**suggestion** (non-blocking): this is a more concise way to get the same thing - which obvious applies to more or less every time you're querying for specific date inputs. Not a big deal though.

Reply:
It's a difficult one to judge tbh, as the way the [docs](https://testing-library.com/docs/queries/about/#priority) are written it suggests that `getByRole` takes priority over the other queries. It does say that `getByLabelText` is ideal for testing form fields but as we're wrapping the inputs in some additional features and functionality I'm leaning towards using `getByRole` to confirm the component is part of the accessibility tree and to remain consistent with the approach we've adopted in other test files.

Reply:
fair enough, it's not a huge thing and on reflection you're probably right anyway 👍